bossdata/spec.py

```diff
@@ -16,6 +16,36 @@
 import astropy.table
 
 
+def get_fiducial_pixel_index(wavelength):
+        """
+        Convert a wavelength to a fiducial pixel index.
+
+        Args:
+            wavelength(float): Input wavelength in Angstroms.
+
+        Returns:
+            numpy.ndarray: Array of floating-point indices relative to the fiducial
+                wavelength grid.
+        """
+        return (np.log10(wavelength) - _fiducial_log10lam0)/_fiducial_coef
+
+_fiducial_coef = 1e-4
+_fiducial_log10lam0 = np.log10(3500.26)
+
+fiducial_pixel_index_range = (0, 4800)
+"""
+Range of fiducial pixel indices covering all spectra.
+
+Pixel indices are calculated by :func:`get_fiducial_pixel_index`.
+"""
+
+fiducial_loglam = (_fiducial_log10lam0 +
+                   _fiducial_coef * np.arange(*fiducial_pixel_index_range))
```

Comment:
The sdss data model uses `coeff0` and `coeff1` for `_fiducial_log10lam0` and `_fiducial_coef`. Perhaps at least worth mentioning the in documentation somewhere.